homeassistant/components/octoprint/camera.py

```diff
@@ -70,3 +108,152 @@ def __init__(
             unique_id=device_id,
             verify_ssl=verify_ssl,
         )
+
+
+class _OctoprintCamera(CoordinatorEntity[OctoprintDataUpdateCoordinator], Camera):
```

Comment:
I followed the existing pattern of inheriting these cameras from `CoordinatorEntity[OctoprintDataUpdateCoordinator]`, but I'm not certain that is actually doing anything useful? They are not using the coordinator in an obvious way, and `OctoprintDataUpdateCoordinator` only seems to query the printer information and not the webcam information, anyways.